docs/manage/fleet/reuse-configuration.md

```diff
@@ -64,6 +64,12 @@ If you want to edit the fragment later, do it from this screen.
 {{% /tablestep %}}
 {{< /table >}}
 
+{{< alert title="Tip: Organize resources into folders" color="tip" >}}
+
+If you have many components and services in one fragment, you can add one or more folders to your fragment and use them to organize the resources.
```

Comment:
```suggestion
If you have many components and services in one fragment, you can add folders to your fragment and use them to organize the resources.
```
[opt] I think they'll understand without the "one or more," and this makes it quicker to read and feels less overly formal. Same below.